### ISSUE TITLE
Syntax error when closing bracket and closing parenthesis are on different lines

### ISSUE THREAD

**Issue by Darkle** [princejwesley/Mancy#64]
I seem to be getting a syntax error `SyntaxError:  missing ) after argument list` with the following code:

``` javascript
Promise.resolve('finished').then(function(result){
  console.log(result)
}
)
```
But if I put the closing bracket and parenthesis on the same line, there's no error:
``` javascript
Promise.resolve('finished').then(function(result){
  console.log(result)
})
```

I'm running Mancy v2.1.0 on OSX 10.11.1.

(thanks for making Mancy btw, it's great)

**Comment by princejwesley** [comment]
@Darkle Can you able to reproduce it consistently?
![image](https://cloud.githubusercontent.com/assets/1622515/11450706/45628f12-95d0-11e5-9eca-084e09dc3a27.png)

**Comment by Darkle** [comment]
Yeah:
![image](https://cloud.githubusercontent.com/assets/6868833/11450718/c9e9ccd6-95fe-11e5-8af5-d779c6387672.png)

Here are my settings:
![image](https://cloud.githubusercontent.com/assets/6868833/11450721/def510fe-95fe-11e5-8687-94a3392d1301.png)

**Comment by Darkle** [comment]
I just tried it with the Babel Transform setting enabled and the error no longer occurs.

**Comment by princejwesley** [comment]
@Darkle thanks, let me look into that

**Comment by princejwesley** [comment]
@Darkle looks like node's repl is throwing same error(Mancy is backed up node's repl)
![image](https://cloud.githubusercontent.com/assets/1622515/11450775/8aaed4c0-95d2-11e5-98f5-6fcef00982b5.png)

@Darkle can you raise ticket in node repository?

**Comment by Darkle** [comment]
Sure thing: https://github.com/nodejs/node/issues/4060

**Comment by princejwesley** [comment]
@Darkle this issue is fixed as part of #79 ticket.
